### PR TITLE
Menu entry clickability

### PR DESCRIPTION
**Pull request by jancborchardt**
Adjust to proper minimum 44px size (we need to take care that all elements have this minimum clickable size), please review @nextcloud/designers 
Also has more breathing space and looks less cramped now. :)

Before:
before

After:

after

### CODE REVIEW

File: core/css/apps.scss [hunk 1]
```diff
@@ -862,6 +862,7 @@ $popovericon-size: 16px;
 			align-items: flex-start;
 			height: auto;
 			margin: 0;
+			padding: 3px 20px 3px 3px !important; /* for nice visual balance */
```
Comment (skjnldsv) [inline core/css/apps.scss]: I'm not fan of using the important elements
Why is this needed?

Comment (skjnldsv) [reply core/css/apps.scss]: Also, far too much padding on buttons
capture d ecran_2018-06-05_16-14-18

**Review by skjnldsv — CHANGES_REQUESTED** [review]
Does not fit all the use cases of the popover menus.

capture d ecran_2018-06-05_16-12-01

**Comment by rullzer** [comment]
Conflicts.

What is left here?

File: core/css/apps.scss (hunk 1 above)
Comment (jancborchardt) [reply core/css/apps.scss]: >  Also, far too much padding on buttons and other elements :)

No? 44px is correct minimum height, and the padding on the right is so it’s nicely balanced with the left where the icon is.

**Comment by jancborchardt** [comment]
> Does not fit all the use cases of the popover menus.

@skjnldsv this doesn’t happen in any app, no?

I would say let’s merge this, and the small discrepancies we can fix in a follow-up.

@rullzer it’s rebased on master. :)

File: core/css/apps.scss (hunk 1 above)
Comment (skjnldsv) [reply core/css/apps.scss]: I thought it was a mistake. It looks nice visually yes :)
Do you remember why you needed the `important` though?
I would prefer if we removed them! ;)

**Comment by skjnldsv** [comment]
> I would say let’s merge this, and the small discrepancies we can fix in a follow-up.

I disagree, I worked hard to fit all those different options in the popover, please don't break them ;)

**Comment by jancborchardt** [comment]
@skjnldsv right – will see to fix this :)

**Comment by MorrisJobke** [comment]
> @jancborchardt Any news on this one?

Ping ... or we need to move this to 15.

File: core/css/apps.scss (hunk 1 above)
Comment (pixelipo) [reply core/css/apps.scss]: I agree with @skjnldsv regarding the `!important` tag - might be a good idea to check why is it actually needed here. Perhaps this rule should be applied higher "upstream"?

**Comment by MorrisJobke** [comment]
15 it is then.

**Comment by AnonymousPolarbearFan** [comment]
Did we make any progress here? I would really appreciate this feature. 👍

**Comment by MorrisJobke** [comment]
I guess we need to move this to 16 😢

**Comment by MorrisJobke** [comment]
#12785 will fix the this in another way - thus closing this one here.

**Pull request closed without merging**